anodot-reporter3/src/main/java/com/kenshoo/metrics/anodot/metrics3/Anodot3ReporterBuilder.java

```diff
@@ -41,9 +47,25 @@ public Anodot3ReporterBuilder turnZeroFilterOff() {
     }
 
     public AnodotReporterWrapper build(MetricRegistry metricRegistry) {
-        return new Anodot3ReporterFactory(conf, globalProperties).anodot3Reporter(metricRegistry);
+        final AnodotMetricFilter filter = composeFiltersIntoOne();
+        return new Anodot3ReporterFactory(conf, globalProperties, filter).anodot3Reporter(metricRegistry);
+    }
 
+    private AnodotMetricFilter composeFiltersIntoOne() {
```

Comment:
Can we avoid this duplication between metrics2 and 3?
I'm guessing this is a pattern we'd be seeing more...